test/dotnet.Tests/TelemetryCommandTest.cs

```diff
@@ -227,6 +229,16 @@ public void InternalreportinstallsuccessCommandCollectExeNameWithEventname()
         }
 
         [Fact]
+        public void TopLevelCommandFailsIfNoValueIsSuppliedAsVerbosity()
```

Comment:
I am not sure this test belongs in a TelemetryCommandTest class.

Reply:
I thought that this was the best location available. Should I create a new file, called TelemetryFilterTests.cs, in the same folder as TelemetryCommandTest.cs, and then move the test to that file?